### PR TITLE
Add unit tests for cdi and jsonp boosters

### PR DESCRIPTION
**Pull request by ajm01**
add unit tests for cdi and jsonp boosters

### CODE REVIEW

File: boost-maven/boost-boms/boost-ee8-apis-bom/pom.xml [hunk 1]
```diff
@@ -36,6 +36,16 @@
                 <artifactId>javax.transaction-api</artifactId>
                 <version>1.2</version>
             </dependency>
+            			<dependency>
+				<groupId>javax.enterprise</groupId>
+	            <artifactId>cdi-api</artifactId>
+				<version>2.0</version>
+			</dependency>
```
Comment (scottkurz) [inline boost-maven/boost-boms/boost-ee8-apis-bom/pom.xml]: Indentation above and below.

**Comment by scottkurz** [comment]
Also as an optional idea, we could consider tracking by expanding [this](https://github.com/OpenLiberty/boost/wiki/Current-Status-of-Boost-Technology-Boosters) chart.

File: boost-maven/boost-boms/boost-ee8-apis-bom/pom.xml (hunk 1 above)
Comment (ajm01) [reply boost-maven/boost-boms/boost-ee8-apis-bom/pom.xml]: it took two of us, but formatting should be right now

**Comment by scottkurz** [comment]
I'm going to cancel the Travis and merge this.. we've run enough tests here

**Pull request merged**